packages/web/src/components/tipping/tip-audio/TipSent.tsx

```diff
@@ -95,12 +94,13 @@ export const TipSent = () => {
       </div>
       <div className={styles.flexCenter}>
         <Button
-          className={styles.shareButton}
-          type={ButtonType.PRIMARY}
-          text={messages.shareToTwitter}
+          variant='primary'
+          color='blue'
```

Comment:
is this the same as the twitter blue color?

Reply:
just confirmed, yes it is!